src/aws/osml/model_runner/tile_worker/tile_worker.py

```diff
@@ -40,15 +40,10 @@ def run(self) -> None:
 
             if image_info is None:
                 logging.info("All images processed. Stopping tile worker.")
-                logging.info(
```

Comment:
Why are we deleting this? In fact we may need to do more with this information. It looks like bot the SMDetector and the new HTTPDetector will log but swallow any errors/exceptions and return an empty FeatureCollection. That is likely the root of the bug where images are completing successfully even when a tile fails to process successfully. 

I assume this was deleted because the new HTTP detector isn't keeping these counts but we should go the other way and have the detector track that information and possibly do something with this result.

Reply:
Added additional logging and retry tracking to address this issue